### DISCUSSION TITLE
Great plugin, should it be maintained as part of the vue-styleguidist monorepo?

### DISCUSSION THREAD

**Discussion by elevatebart** [andrewcourtice/vite-plugin-vue-docgen#3] [Q&A]
Hello @andrewcourtice 

I was about to create it, but you beat me to the point.
I am going to be integrating this plugin in https://github.com/histoire-dev/histoire
I just wanted to make sure that it could be maintained for the long run.

Hit me up on [discord](https://chat.vuejs.org/) or Twitter. if you wanna chat.

**Comment by andrewcourtice** [comment]
Hi @elevatebart 

That's great 😄  Histoire is an awesome project!

I plan on maintaining this project into the future, however I'm not particularly focused on adding features to it. It's really just meant to be a thin wrapper around the vue-docgen-api. Ideally this would eventually be part of the vue-styleguidist repo but it's up to the maintainers of that project as to whether they want to take on this plugin.

In short - yes I plan on maintaining this project into the future so you are safe to use it and expect fixes to bugs. I will contact you or post an announcement on the README if I ever plan on discontinuing maintenance of this project.

If there are features you need, you are more than welcome to contribute to the project 😄 

Thanks,
Andrew

**Comment by elevatebart** [comment]
> it's up to the maintainers of that project as to whether they want to take on this plugin.

hehehe 😄

I find it flattering that you think there are multiple maintainers of vue-styleguidist. It's just me for now.

More seriously, I don't think we need to change a thing for now. 
I was simply checking in, if things become incompatible with vite 3 for instance I would probably fix those issues and make a PR.
